network-functions/core-network/charts/astri-upf/values.yaml

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2021 Astri Corporation
+
+# Default values for upf.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+upf:
+  replicaCount: 1 
+  image:
+    repository: 192.168.205.17/5g/upf
+    tag: 1.0.0
+    pullPolicy: IfNotPresent
+
+  # devices: A list of string - phisical functions, in sysfs pci address format. eg: device : 0000:09:00.0
+  # Domain:Bus:Device:Function 
+  # type: string - "netdevice"|"vfio"
+  networks:
+    - name: networkn-upf
+    - name: NWI0-N3N9
+      device: 0000:b7:00.0
+      # rate of the above device, unit Gbps. Only support 1, 10, 25, 40, 100
+      rate: 10
+    - name: NWI0-N6
+      device: 0000:b7:00.1
+      # rate of the above device, unit Gbps. Only support 1, 10, 25, 40, 100
+      rate: 10
+
+  N3IP: 192.168.120.51 
```

Comment:
Please avoid using the segment 192.168.0.0/16 as this is getting conflicted in many cases with server IPs, since most of the wifi routers by default configured with these segment of IPs. Instead use something like 193.x.x.x